### ISSUE TITLE
Results are not sorted

### ISSUE THREAD

**Issue by gpgekko** [webjars/webjars#1598]
Minor inconvenience, but search results are not sorted. I searched for `@material` to update a bunch of packages in that namespace, and they came in random order. It would be nice to have them sorted alphabetically.

**Comment by jamesward** [comment]
It is actually sorted by popularity.  But that isn't exposed.  So I wonder if search should always sort by name or if sorting should be user configurable.  WDYT?

**Comment by gpgekko** [comment]
Yeah, that is quite hard to tell from the UI. I think user configurable would be great, but I'm guessing that will add quite a bit of complexity. Personally I would prefer alphabetically in most search results situations, but there is something to say for the current approach too.
A quick thing to prevent confusion for now would be to append a small 'sorted by popularity' line to the 'Search Results' header.
